src/App.js

```diff
@@ -1,5 +1,5 @@
-import React, { Component, Suspense } from 'react'
-import { HashRouter, BrowserRouter, Route, Routes, Navigate } from 'react-router-dom'
```

Comment:
gonna need hash router for swa at some point

Reply:
Would you say leave out for now - or re-add it when/if required?